ouroboros-consensus/src/Ouroboros/Consensus/Storage/LedgerDB/OnDisk.hs

```diff
@@ -158,6 +164,32 @@ data InitLog blk =
   | InitFailure DiskSnapshot (InitFailure blk) (InitLog blk)
   deriving (Show, Eq, Generic)
 
+{-------------------------------------------------------------------------------
+  TraceReplayEvent decorator
+-------------------------------------------------------------------------------}
+
+-- | 'TraceReplayEvent' instantiated with additional information.
+--
+-- The @replayTo@ parameter is instantiated with the 'Point' of
+-- the tip of the ImmutableDB.
+type TraceLedgerReplayEvent blk = TraceReplayEvent blk (Point blk, Point blk)
+
+-- | Add the tip of the Immutable DB to the trace event
+--
+-- Between the tip of the immutable DB and the point of the starting block,
+-- the node could (if it so desired) easily compute a "percentage complete".
+decorateReplayTracerWithGoal
```

Comment:
Please change this to 

```
  :: ReplayGoal blk
  -> Tracer m (TraceReplayEvent blk (ReplayStart blk, ReplayGoal blk))
  -> Tracer m (TraceReplayEvent blk (ReplayStart blk)
```

or -- if we go with my new suggestion -- to:

```
  :: ReplayGoal blk
  -> Tracer m (TraceReplayEvent blk)
  -> Tracer m (ReplayGoal blk -> TraceReplayEvent blk)
```

Reply:
Done